test/gen_integration.sh

```diff
@@ -1,9 +1,11 @@
+export WORKFLOWS_PATH=.github/workflows
```

Comment:
💡🥷☕️ Please note: Probable duplication, if this PR is doing same this as As (@aamgayle ) a has done then please co-ordinate with him so that his PR gets merged first: https://github.com/Azure/draft/pull/204/files#diff-2232de3c545951fb34c43095b1f979a7c86e41f771e25438d621f9aebde667c7R1 

Thanks,